### ISSUE TITLE
Issues with 16x10 phones, weird scenario issue. 

### ISSUE THREAD

**Issue by Theseventensplit** [scrubbless/PublishedScenarios#21]
Hi, great mystery so far, but we had to restart. For two reasons:
1: my phone is a new Samsung s10e. It has an extra wide screen, and this seems to mess with valkyrie. When we open the journal the exit button is not visible. There is no way to continue and the game had to be closed.
2: we encountered issues where, when visitor time started, it had us remove our players from the board and place them in the lounge, even though it wouldn't make sense with the narrative for three of them to be escorted there (one had been teleported to the library, the person who had been interred there had been teleporged to the basement. And another had snuck upstairs and was still there. But there was no clear reason why the orderly Eric would have rounded them up, as he didn't know they were there.

**Comment by scrubbless** [comment]
Thanks for your feedback and sorry for my late reply!

Which scenario were you playing? Valkyrie has two types of scaling "horizontal and vertical" with the default being horizontal.
I didn't realize this scaling issue until after I finished a strain on reality, and it's now too much work to fix (every age would need realignment). I have an s9+ and noticed the same issue.
I will try to move the buttons however, they may clip text on extra wide screen devices, but it may be the lesser of evils.

I've since switched to vertical scaling for every ui element I create to avoid this issue.
